src/Components/CreatePoiForm.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { connect } from 'react-redux';
+import './ComponentsCSS/createPoiForm.css';
+import axios from 'axios';
+
+const mapStateToProps = state => ({
+  name: state.name,
+  keywordOne: state.keywordOne,
+  keywordTwo: state.keywordTwo,
+  geolocCoordonnees: state.geolocCoordonnees,
+  poiKeywordsDisplay: state.poiKeywordsDisplay,
+});
+
+// get date and time to fill creation_date field in database
+const poiCreationDate = new Date().toISOString().slice(0, 19).replace('T', ' ');
+
+const CreatePoiForm = ({
+  dispatch, name, geolocCoordonnees, poiKeywordsDisplay,
+}) => (
+  // eslint-disable-next-line no-unused-expressions
+  <div>
+    <form
+      className="poi-create"
+      method="POST"
+      encType="multipart/form-data"
+      onSubmit={(e) => {
+        axios.post('http://localhost:3001/pois', {
+          name,
+          latitude: geolocCoordonnees[0],
+          longitude: geolocCoordonnees[1],
+          creation_date: poiCreationDate,
+          author_id: 1,
+        })
+          .then(res => console.log(`${res} has been properly sent`))
+          .catch(err => console.log(err));
+        e.preventDefault();
+      }
+    }
+    >
+
+      <legend>Ajoutez un point d'intérêt</legend>
+      <div className="poi-address">
+        <p>
+          Adresse:
+          <span> en fonction des coordonnées gps</span>
+        </p>
+      </div>
+
+
+      <div className="poi-name">
+        <label htmlFor="name">Nom</label>
+        <br />
+        <input
+          type="text"
+          id="name"
+          name="name"
+          onChange={e => dispatch({ type: 'HANDLE_FORM_NAME_CHANGE', name: e.target.value })}
+          required
+        />
+      </div>
+      <div>
+        <label htmlFor="keywordOne">Catégorie:</label>
+        <br />
+        <select id="keywordOne" name="keywordOne" onChange={e => dispatch({ type: 'HANDLE_FORM_K1_CHANGE', keywordOne: e.target.value })} required>
+          {poiKeywordsDisplay.filter(keyword => keyword.importance === 1)
+            .map(keyword => (
+              <option key={keyword.name} id={keyword.name} value={keyword.name}>
+                {(keyword.name)}
+              </option>
+            ))}
+        </select>
+      </div>
+
+      <div>
+        <label htmlFor="keywordTwo">Sous-catégorie (facultatif):</label>
+        <br />
+        <select id="keywordTwo" name="keywordTwo" onChange={e => dispatch({ type: 'HANDLE_FORM_K2_CHANGE', keywordTwo: e.target.value })}>
+          {poiKeywordsDisplay.filter(keyword => keyword.importance === 2)
```

Comment:
Don't you have to filter the second point of interest accordingly to the first one ? To propose only the ones which are linked to the first one ? Like in Sport you will have ping pong table, football fields, swimming pool, etc ... and not parc, bins, etc ...